rules/TypeDeclaration/Rector/StmtsAwareInterface/IncreaseDeclareStrictTypesRector.php

```diff
@@ -101,7 +101,7 @@ public function beforeTraverse(array $nodes): ?array
         $rectorWithLineChange = new RectorWithLineChange(self::class, $stmt->getLine());
         $this->file->addRectorClassWithLine($rectorWithLineChange);
 
-        return \array_merge([$strictTypesDeclare, new Nop()], $nodes);
+        return [$strictTypesDeclare, new Nop(), ...$nodes];
```

Comment:
This rule should be excluded, as it makes code harder to read :+1:

Reply:
sure, excluded https://github.com/rectorphp/rector-src/pull/5924/commits/fa0e1d48d4dc53359806080effb1063f998ea95c